gsa/src/web/pages/alerts/details.js

```diff
@@ -147,10 +147,10 @@ const AlertDetails = ({capabilities, entity, links = true}) => {
 
           {tasks.length > 0 && (
             <TableRow>
-              <TableData>{_('Task using this Alert')}</TableData>
+              <TableData>{_('Tasks using this Alert')}</TableData>
               <TableData>
                 <HorizontalSep wrap>
-                  {tasks.map(task => (
+                  {tasks.map((task, index) => (
```

Comment:
Why did you add the unused index here?

Reply:
It was an accident... I forgot to remove it.